app/models/activity.rb

```diff
@@ -26,6 +26,10 @@ def project=(new_project); end
     includes(:trackable).order("activities.created_at DESC").limit(20)
   end
 
+  scope :all_latest, -> do
+    includes(:trackable).order("activities.created_at DESC")
```

Comment:
To deal with the date issue we can actually perform a group_by `group_by { |item| item.created_at }` which will separate activity items into groups by date.

The above code likely needs fine tuning as it may try to sort down to the second, instead of just the Day.

Reply:
@sean-yeoh I believe this could be removed now, correct?

Reply:
Yeah, i've removed it